### PR TITLE
Adds build+deploy scripts for non-IDE setup

### PR DESCRIPTION
**Pull request by sudocurse**
Creates scripted build & deploy and adds docs.

### CODE REVIEW

**Comment by sudocurse** [comment]
I would love some careful eyes on this as it does fun things like blindly copy & delete files and system services on a remote device as root 😅. Specifically `build.sh` which makes and then puts everything into "result/" and `draft-rm-install` which copies the entire result/ directory into the device root dir.

File: build.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+TOOLCHAIN=rm-toolchain
+source $TOOLCHAIN/environment-setup-cortexa9hf-neon-oe-linux-gnueabi && \
```
Comment (Eeems) [inline build.sh]: Any reason you are using `&&` instead of just running `set -e` at the start like you do in `draft-rm-install` and `draft-rm-uninstall`?

Comment (sudocurse) [reply build.sh]: Good point, this is messy. Fixed and added `set -x` too.

**Comment by sudocurse** [comment]
This PR got a little messy, making a fresh one

**Pull request closed without merging**